base/linalg/cholesky.jl

```diff
@@ -249,7 +249,7 @@ end
 
 ### Default to no pivoting (and storing of upper factor) when not explicit
 cholfact!(A::Hermitian) = cholfact!(A, Val{false})
-cholfact!{T<:Real,S}(A::Symmetric{T,S}) = cholfact!(A, Val{false})
+cholfact!(A::Symmetric{<:Real}) = cholfact!(A, Val{false})
```

Comment:
Can this really go from two parameters to one? I guess the 2nd isn't used at all?